src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -93,7 +93,10 @@
     }
 };
 
-if (PHP_VERSION_ID >= 70200) {
+if (PHP_VERSION_ID >= 70300) {
```

Comment:
this is a BC break (people using the bridge will be upgraded to the PHPUnit 9 version, with all its BC breaks impacting the testsuite) in the LTS version of Symfony. that's not good. this PR must go in master.

Reply:
Is it better to remove this condition to let everyone have the option? Or maybe to push it to master in a sperate PR?

I think the first part of this PR is enough for the compatibility case PHPUnit9/SF4.4

Reply:
Splitting might be a good idea, the other changes in the PR are OK for 4.4
But looking at the changelog, I'm not sure we need to bother ppl with a bump of the default version honeslty.

Reply:
PR is updated. 

I removed the bump of the default version, to make this PR compatible with 4.4 branch.

Let me know if you want a PR to master that enable the default version.

Thanks, both of you for your feedback.